packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1843,10 +1843,19 @@ export class ProjectService {
             description: 'Gets all filters available for a single query',
         });
         try {
-            const savedChart = await this.savedChartModel.get(savedChartUuid);
```

Comment:
why not using the existing chartSummary ?

Reply:
I thought about that first, but `getChartSummaryQuery` gets extra stuff we don't need, accessing tables for `PinnedChartTableName` and doesn't `innerJoin` `saved_queries_versions` table (to get the `explore_name`), so to avoid impacting that query, I've decided to create a different one

Reply:
`getSummary` vs. `get`:
The GetSummary average request time of 6.6 ms is an improvement of approximately 97.93% over the Get average request time of 318.6 ms.

`getInfoForAvailableFilters` vs. `get`:
The GetInfoForAvailableFilters average request time of 135 ms is an improvement of approximately 57.62% over the Get average request time of 318.6 ms.

NOTE: `getSummary` doesn't request `saved query versions`, so this query is missing the `explore_name` that is necessary to get `availableFilters`

This was done on: https://lightdash-pr-7132.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/dashboards/512f84dc-bacf-4a2b-9591-1f2079db36f3/view?filters=%7B%22dimensions%22%3A%5B%7B%22id%22%3A%22e7df7c5a-1070-439a-8300-125fe5f9b1af%22%2C%22target%22%3A%7B%22fieldId%22%3A%22orders_is_completed%22%2C%22tableName%22%3A%22orders%22%7D%2C%22values%22%3A%5Btrue%5D%2C%22operator%22%3A%22equals%22%7D%2C%7B%22id%22%3A%226d28a3a5-c01f-46d8-9f6b-74a9ab1efd99%22%2C%22target%22%3A%7B%22fieldId%22%3A%22orders_order_date_year%22%2C%22tableName%22%3A%22orders%22%7D%2C%22values%22%3A%5B5%5D%2C%22operator%22%3A%22inThePast%22%2C%22settings%22%3A%7B%22completed%22%3Atrue%2C%22unitOfTime%22%3A%22years%22%7D%7D%5D%2C%22metrics%22%3A%5B%5D%7D
Refresh Dashboard and see logs: https://dashboard.render.com/web/srv-ck4t9poj9kis73969hjg/logs?mode=maximize

Reply:
Ok, now with another method that gets the summary AND the exploreName: 


#### `get`
- get for chart bb40529b-28fb-442c-b07a-495c5521191e took 298 ms
- get for chart 36966404-dd2e-499f-902e-6ffb78900072 took 392 ms
- get for chart 0e26f734-ec36-4c7b-8d0f-903e1ad65cf0 took 489 ms
- get for chart d4ec62ba-eb97-4247-a7fb-6751ee79b19b took 493 ms
- get for chart c4e6db63-ccc0-475d-afdc-5619a98c2385 took 588 ms

#### `getSummary`
- getSummary for chart 36966404-dd2e-499f-902e-6ffb78900072 took 205 ms
- getSummary for chart 0e26f734-ec36-4c7b-8d0f-903e1ad65cf0 took 109 ms
- getSummary for chart c4e6db63-ccc0-475d-afdc-5619a98c2385 took 103 ms
- getSummary for chart bb40529b-28fb-442c-b07a-495c5521191e took 396 ms
- getSummary for chart d4ec62ba-eb97-4247-a7fb-6751ee79b19b took 299 ms

#### `getSummaryWithExploreName`
- getSummaryWithExploreName for chart 36966404-dd2e-499f-902e-6ffb78900072 took 102 ms
- getSummaryWithExploreName for chart c4e6db63-ccc0-475d-afdc-5619a98c2385 took 99 ms
- getSummaryWithExploreName for chart d4ec62ba-eb97-4247-a7fb-6751ee79b19b took 102 ms
- getSummaryWithExploreName for chart 0e26f734-ec36-4c7b-8d0f-903e1ad65cf0 took 99 ms
- getSummaryWithExploreName for chart bb40529b-28fb-442c-b07a-495c5521191e took 293 ms

#### `getInfoForAvailableFilters`
- getInfoForAvailableFilters for chart 36966404-dd2e-499f-902e-6ffb78900072 took 94 ms
- getInfoForAvailableFilters for chart c4e6db63-ccc0-475d-afdc-5619a98c2385 took 103 ms
- getInfoForAvailableFilters for chart d4ec62ba-eb97-4247-a7fb-6751ee79b19b took 4 ms
- getInfoForAvailableFilters for chart 0e26f734-ec36-4c7b-8d0f-903e1ad65cf0 took 298 ms
- getInfoForAvailableFilters for chart bb40529b-28fb-442c-b07a-495c5521191e took 304 ms

### Average Time for Each Request Type
- `get`: 452 ms
- `getSummary`: 222.4 ms
- `getSummaryWithExploreName`: 139 ms
- `getInfoForAvailableFilters`: 160.6 ms

### Percentage Faster Compared to `get`
- `getSummaryWithExploreName`: 69.25%
- `getInfoForAvailableFilters`: 64.49%

Reply:
Good info 🙌  imo we should go with `getInfoForAvailableFilters`.

Reply:
agree @ZeRego , just removed the tracking and kept only the introduction of `getInfoForAvailableFilters`